src/web/app/components/question-responses/per-question-view-responses/per-question-view-responses.component.html

```diff
@@ -51,13 +51,15 @@
       <tr *ngFor="let response of responsesToShow" [class.color-neutral]="response.isMissingResponse">
         <td *ngIf="showGiver">{{ response.giverTeam }}</td>
         <td *ngIf="showGiver">
-          {{ response.giver === '-' ? 'No Specific User' : response.giver }}
+          {{ response.giver === '-' ? 'No Specific User' : response.giver}}
+          {{response.giverEmail? " (" + response.giverEmail + ")" : ""}}
         </td>
         <td *ngIf="showRecipient">
           {{ response.recipientTeam === '-' ? 'No Specific Team' : response.recipientTeam}}
         </td>
         <td *ngIf="showRecipient">
-          {{ response.recipient === '-' ? 'No Specific User' : response.recipient }}
+          {{ response.recipient === '-' ? 'No Specific User' : response.recipient}}
+          {{response.recipientEmail? " (" + response.recipientEmail + ")" : ""}}
```

Comment:
not sure if I am correct, angular template doesn't seem to allow template literals. 
https://stackoverflow.com/questions/47057696/how-to-use-es6-template-literal-as-angular-component-input
i couldn't use {{...}} string interpolation again as it is used in the outer conditional expression

Reply:
Yup I think you're right, ES6 template literals and nested string interpolation aren't supported. Could probably 1) move it out into a function then call it or 2) create a separate element for it with:
`<ng-container *ngIf="response.recipientEmail">({{ response.recipientEmail }})</ng-container>`

Leaning towards the latter, but tbh I don't think we need to go so far to avoid string concatenation. What do you all think?

Reply:
The simple solution will suffice here, shouldn't overcomplicate things.

Reply:
Okay! Resolving this convo then